.github/workflows/publish.yml

```
@@ -0,0 +1,218 @@
+name: Publish Docker image, Helm chart and Wire build
+on:
+  push:
+    branches: [master, dev, edge, stefan]
```

should we remove the stefan branch also if this is finalized? 
```suggestion
    branches: [master, dev, edge]
```

---

I removed it